geneplexus/download.py

```diff
@@ -186,8 +186,10 @@ def _make_download_options_list(
 
     for i in opts:
         if i not in check_list:
-            raise ValueError(f"Unexpected {name}: {opts!r}")
-
+            raise ValueError(
+                f"Unexpected {name}: {opts!r} when downloading\n"
+                + "If using custom networks/GSCs turn auto_download off",
```

Comment:
Could you elaborate about this note a little more (as well as #175)? I think if a user were to use custom networks, they are not going to try to "download" that network anyways, so why is this necessary?